stubs/docs/build.gradle.kts

```diff
@@ -33,7 +32,7 @@ val GOPATH = extra["gopath"]
 
 protobuf {
     protoc {
-        artifact.set("com.google.protobuf:protoc:3.6.1")
+        artifact.set("com.google.protobuf:protoc:3.11.1")
```

Comment:
Does it fail without explicitly setting this?

Reply:
Thought the was causing javadoc errors at first (since there was that one version of protoc that was broken), but works fine with 3.6.1. Rolled back all instances.